public/app/plugins/panel/timeseries/NullsThresholdInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { rangeUtil } from '@grafana/data';
+import { Input } from '@grafana/ui';
+
+export enum InputPrefix {
+  LessThan = 'lessthan',
+  GreaterThan = 'greaterthan',
+}
+
+type Props = { value: number; onChange: (value?: number | boolean | undefined) => void; inputPrefix?: InputPrefix };
+
+export const NullsThresholdInput = ({ value, onChange, inputPrefix }: Props) => {
```

Comment:
should rename since nothing in here is specific to `null`, it just collects a time duration string, and may in the future collect a numeric delta (for trend).

maybe DeltaThresholdInput?

even the prefix is optional, which means it may just be `DeltaWithPrefixInput`?